src/app/courses/components/courses-list/courses-list.component.ts

```diff
@@ -35,16 +44,19 @@ export class CoursesListComponent implements OnInit {
           'Learn about where you can find course descriptions, what information they include, how they work, and details about various components of a course description. Course descriptions report information about a university or colleges classes. Theyre published both in course catalogs that outline degree requirements and in course schedules that contain descriptions for all courses offered during a particular semester.',
         duration: 48,
         creationDate: new Date(),
```

Comment:
Could you please change the date to show the course without any border? It's one of the requirements.